datadog_checks_dev/datadog_checks/dev/tooling/commands/dep.py

```diff
@@ -148,7 +148,11 @@ def sync():
 def filter_releases(releases):
     filtered_releases = []
     for version, artifacts in releases.items():
-        parsed_version = parse_version(version)
+        try:
+            parsed_version = Version(version)
+        except InvalidVersion:
+            continue
```

Comment:
Nit: It may be useful to output something here if the version parsing failed? Not sure how common this will be.

Reply:
<details>

```
❯ ddev dep updates --sync
0.8d
0.9d
1.5d
1.8d
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.4.*, <4
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
>=3.6.*
0.1-bulbasaur
0.1-charmander
0.9-doduo
0.9-eevee
0.9-fearow
0.9-gyarados
0.9-horsea
0.9-ivysaur
3.x
4.1.16d
5.5.0-2014.1
5.5.0-2014.1.1
0.1.0.1.g46ab6a7
a3
Files updated: 29
Updated 22 dependencies
```
</details>

Reply:
Talked offline, and the information is not that useful.